src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -99,7 +95,9 @@ private void checkIndex(int index) {
         }
     }
 
-    private boolean checkSize() {
-        return size == elements.length;
+    private void checkNgrow() {
```

Comment:
I will approve but you`ll better change the method name because other reviewers will obviously doesn't like your name :)